pkg/graveler/sstable/range_manager.go

```diff
@@ -92,7 +92,13 @@ func (m *RangeManager) GetValueGE(ctx context.Context, ns committed.Namespace, i
 		}
 		return nil, ErrKeyNotFound
 	}
-	vBytes, _, err := value.Value(nil)
+	vBytes, owned, err := value.Value(nil)
+	if !owned {
```

Comment:
Wow, great catch!

Is there any way for us to _change_ this bad ownership status?  IIUC when `!owned`, iterator `it` shares underlying state with some other iterator.  I would expect this state to persist, and now very many calls (including future `getValue()`s) will keep on copying every value.  It might be considerably more efficient if we could convince Pebble to unshare the iterator -- say by creating future iterators on a _new_ iterator hierarchy.  (**It goes without saying** that this is no reason to delay this PR!)